ckan/templates/package/snippets/stages.html

```diff
@@ -20,9 +20,9 @@
 <ol class="stages {{ class }}">
   <li class="first {{ s1 }}">
     {% if s1 != 'complete' %}
-      <span class="highlight">{{ _('Create dataset') }}</span>
+      <span class="highlight">{{ _('Create ' ~ dataset_type) }}</span>
```

Comment:
You used `.title()` above, what's the difference here?

Reply:
I'm just trying to leave labels as they were before. In previous example **Dataset** was originally in title-case and here it was in lower-case before my change